### PR TITLE
chore: add variables for keeping cross service requests server internal

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -268,6 +268,7 @@ camac_python_pyenv_virtualenv_path: "{{ camac_python_pyenv_install_directory }}/
 
 # Url of the Camac Django API
 camac_python_url: "http://camac-ng.local/"
+camac_internal_url: "https://localhost"
```
Comment (sbor23) [inline defaults/main.yml]: Are you sure https works on localhost?

Comment (anehx) [reply defaults/main.yml]: Why shouldn't it? This request will be handled by apache which has https configured

Comment (sbor23) [reply defaults/main.yml]: I just wonder if the cert is valid for the `localhost` domain...

Comment (anehx) [reply defaults/main.yml]: You're absolutely right.. I have to configure the URL of the uwsgi service directly..

**Review by sbor23 — APPROVED** [review]

**Pull request merged**